src/routes/app.css

```
@@ -304,6 +304,22 @@
+ * Board load — see docs/design-system §7. Cards fade up 8px at scale .98;
+ * the stagger itself (30ms per card, capped at 8) is applied per-card as an
+ * animation-delay, since that value is dynamic and a keyframe can't express it.
+ */
+@keyframes card-enter {
+	from {
+		opacity: 0;
+		transform: translateY(8px) scale(0.98);
+	}
+}
+
+@utility animate-card-enter {
```

Stylelint (17.14.1)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)